### DISCUSSION TITLE
Consequences of disabling Steno

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#3391] [Unsupported Versions] [answered]
https://docs.securityonion.net/en/2.3/stenographer.html#disabling

Curious what exactly the consequences of this would be from a distributed SO perspective. Specifically, what are the pros/cons? I ask because I might like to store raw pcap (as opposed to .steno files and the associated indexes) on a SAN eventually, but would hate to break any SO functionality. While Steno brings some great things to the table, it has the potential to complicate storing and retrieving data on/from long term storage, albeit likely only slightly.

I am currently trying to absorb the documentation at the following links, but I would find some guidance from the SO community immensely helpful. 
https://github.com/google/stenographer
https://github.com/google/stenographer/blob/master/DESIGN.md

Perhaps this isn't quite the right question, but any input is appreciated.

**Comment by dougburks** [comment, marked as answer]
If you disable Steno, then you can no longer pivot to full packet capture.

It seems like you might be concerned about requesting raw pcap from steno via the command line and so we've updated our documentation with instructions on how to do that:
https://docs.securityonion.net/en/2.3/stenographer.html#analysis

Keep in mind that this is much faster than traditional pcap retrieval since it is indexed by Steno.
